dev_guide/app_tutorials/jenkins_pipeline.adoc

```diff
@@ -0,0 +1,219 @@
+[[dev-guide-jenkins-pipeline-builds]]
+= Jenkins Pipeline Builds
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[introduction]]
+
+== Introduction
+
+Whether you are creating a simple website or a complex web of microservices, it's quick
+and easy to use Jenkins Pipelines to build, test, deploy, and promote your applications on OpenShift.
```

Comment:
easy to use OpenShift Pipelines to build, test.......blah blah.  OpenShift Pipelines provide an OpenShift experience around Jenkins Pipelines.  (and now you can go talk about jenkins pipelines all you want)